### ISSUE TITLE
Layout change for reduced diags resolution on panan0025?

### ISSUE THREAD

**Issue by willaguiar** [COSIMA/mom6-panan#50]
Hello Everyone

To save space for an extended panan-0025 run, we are setting the model to save the diagnostics in half-resolution (FYI, adding `_d2` to the end of the [module name](https://bb.cgd.ucar.edu/cesm/threads/saving-mom6-diagnostics-with-half-horizontal-resolution.8662/#post-51034)).

However for this option to work we need to have a proportional layout:

`FATAL from PE   0: downsample_diag_indices_get: Non-commensurate downsampled domain is not supported. Please choose a layout such that NIGLOBAL/Layout_X and NJGLOBAL/Layout_Y are both divisible by dl=      2 Current domain extents:      5     69      5     61`

Is it possible for us to alter panan-0025 layout for future runs ( either go back to previous one, or set a new one?)

**Comment by micaeljtoliveira** [comment]
@willaguiar It should be possible to change the layout in the required way, although it might require reducing the number of cores and/or using a less efficient IO layout.

I'll see what I can come up with.

**Comment by micaeljtoliveira** [comment]
@willaguiar I was now looking at this more in detail and I don't think we can't do it without modifying the grid. The problem is that we have `NJGLOBAL = 3383` and 3383 is only divisible by 17 and 199.

Maybe we could extend the grid south by one point to have `NJGLOBAL = 3384` , but that requires some extra work.

**Comment by adele-morrison** [comment]
Thanks for looking into this Micael. Maybe easiest if we leave this as is for now, but we should remember this when designing the  OM3 layouts at high res (0.1deg and higher).
@aekiss

**Comment by willaguiar** [comment]
I guess the aim of changing the layout would be to be able to extend the panan-0025 RYF run we currently have, saving the output in a coarser resolution (using smaller disk space). It would take some time for me to set up the new grid, so I think we should maintain RYF as it currently is, but if there are further plans to run other simulations with panan0025, maybe it is worth to set up a new config with the extended domain?
What do you think @adele-morrison ?

**Comment by adele-morrison** [comment]
Yep, agreed, let's leave the existing panan runs as they are, as it would be pretty confusing changing the layout half way through the simulation. 

@pedrocol is working on developing a panan config with ice shelf cavities, and will need to extend the grid south for that. So worth making sure the grid divides evenly for that new config Pedro.

**Comment by aekiss** [comment]
Could also consider saving space by post-processing to reduce numerical precision in the outputs.

**Comment by aekiss** [comment]
Eg https://github.com/COSIMA/01deg_jra55_iaf/blob/01deg_jra55v140_iaf_cycle4/reduce_sigfig.sh
